### ISSUE TITLE
Use `(prefers-reduced-motion: no-preference)` by default

### ISSUE THREAD

**Issue by charlestati** [twbs/bootstrap#37122]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

Currently, Bootstrap sets the `transition` value if transitions are enabled and then sets `transition: none;` in `(prefers-reduced-motion: reduce)`:

```scss
@if $enable-transitions {
  @if nth($transition, 1) != null {
    transition: $transition;
  }

  @if $enable-reduced-motion and nth($transition, 1) != null and nth($transition, 1) != none {
    @media (prefers-reduced-motion: reduce) {
      transition: none;
    }
  }
}
```

Instead, transitions should be set in `(prefers-reduced-motion: no-preference)`.

```scss
@if $enable-transitions {
  @if nth($transition, 1) != null {
    @if $enable-reduced-motion and nth($transition, 1) != none {
      @media (prefers-reduced-motion: no-preference) {
        transition: $transition;
      }
    } @else {
      transition: $transition;
    }
  }
}
```

### Motivation and context

The `prefers-reduced-motion` media query has [94.76 % global support](https://caniuse.com/?search=prefers-reduced-motion).
If someone uses a browser that does not support this media query, they will see the transitions.
It is preferable that websites are accessible by default, which means that animations should be enabled when the browser indicates `no-preference`.

This is open for debate because perhaps some browsers do not set `prefers-reduced-motion` to `no-preference` by default.

**Comment by patrickhlauke** [comment]
in essence, this https://codepen.io/patrickhlauke/pen/YzPPdeo

**Comment by patrickhlauke** [comment]
i've actually just given it a try in a branch, but the changes (to `_progress.scss`, `_spinners.scss`, and `_transition.scss`) end up being a bit more annoyingly repetitive than the suggested change above. taking the example in the OP, it would need to be (unless I'm mistaken)

```scss
@if $enable-transitions {
  @if nth($transition, 1) != null {
    transition: $transition;
    @if $enable-reduced-motion and nth($transition, 1) != none {
      transition: none;
      @media (prefers-reduced-motion: no-preference) {
       transition: $transition;
      }
    }
  }
}
```

similarly, in the others, you get the same dance, a la

```scss
@if $enable-transitions {
  .progress-bar-animated {
    animation: $progress-bar-animation-timing progress-bar-stripes;

    @if $enable-reduced-motion {
      animation: none;
      @media (prefers-reduced-motion: no-preference) {
        animation: $progress-bar-animation-timing progress-bar-stripes;
      }
    }
  }
}
```

```scss
@if $enable-reduced-motion {
  .spinner-border,
  .spinner-grow {
    --#{$prefix}spinner-animation-speed: #{$spinner-animation-speed * 2};
    @media (prefers-reduced-motion: no-preference) {
      --#{$prefix}spinner-animation-speed: #{$spinner-animation-speed};
    }
  }
}
```

```

considering that IE11 has now been officially sunsetted by MS for ages, that most browsers support the media query, and that users that DO require reduced motion at this point will likely transition to using a browser that actually provides them with this option that they need, I'd say the extra "define, then undefine, then redefine" dance required here is unnecessary and makes the SCSS more difficult to understand.
